core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -100,11 +100,18 @@ private void saveDlpPositiveItem(Node node) throws RepositoryException {
     DlpPositiveItemService dlpPositiveItemService = CommonsUtils.getService(DlpPositiveItemService.class);
     DlpPositiveItemEntity dlpPositiveItemEntity = new DlpPositiveItemEntity();
     dlpPositiveItemEntity.setReference(node.getUUID());
+    dlpPositiveItemEntity.setTitle(node.getName());
+    if (node.hasProperty("exo:owner")) {
+      String author = node.getProperty("exo:owner").getString();
+      if ("__system".equals(author)) {
+        return;
```

Comment:
Why do you return is the author is system ?
should we just continue to execute what we have after the if ?

Reply:
I added this case because we have system files that are added to the dlp quarantine when the server starts up, I think they shouldn't.